Assets/MixedRealityToolkit.Tests/PlayModeTests/InputEventSystemTests.cs

```diff
@@ -25,10 +23,16 @@ namespace Microsoft.MixedReality.Toolkit.Tests
 
     class InputEventSystemTests
     {
+        [SetUp]
+        public void Setup()
+        {
+            PlayModeTestUtilities.Setup();
```

Comment:
SetupMrtkWithoutGlobalInputHandlers was using editor-only scene creation so it still needs to be modified. Since it starts by initializing an MRTK with a default profile, the 'standard' setup will still work fine. I'll just have it modify the instance created by the standard setup.